### ISSUE TITLE
vs2019 no context menu options

### ISSUE THREAD

**Issue by pajama** [SaschaWillems/SPIRV-VSExtension#6]
Extension says it is installed. 
vs2019
windows 10

I've tried right clicking on a file called flow.frag, and the containing folder. Are there extra step necessary to recognizing the shader files? Thank you!

**Comment by SaschaWillems** [comment]
No, there are no extra steps required aside from installing the extension. It should detect file extensions and show compile options.

**Comment by RenderLjy** [comment]
vs professional 2019
windows 11
like

> Extension says it is installed. vs2019 windows 10
> 
> I've tried right clicking on a file called flow.frag, and the containing folder. Are there extra step necessary to recognizing the shader files? Thank you!


me too， can not show compile options.
vs professional 2019
windows 11

**Comment by RenderLjy** [comment]
> No, there are no extra steps required aside from installing the extension. It should detect file extensions and show compile options.
Extension says it is installed.
vs2019
windows 11
![image](https://github.com/SaschaWillems/SPIRV-VSExtension/assets/57518985/4566838e-24ce-4af8-9a8b-21e20bcc98ac)
no  compile options. can you help me, Thanks!

**Comment by RenderLjy** [comment]
![image](https://github.com/SaschaWillems/SPIRV-VSExtension/assets/57518985/1f731504-3ca5-4e2a-84b8-fe981d74b7d7)
installed directly from the IDE.
